### ISSUE TITLE
Login button doesn't do anything on iOS

### ISSUE THREAD

**Issue by maxal1917** [xamarin/FacebookComponents#157]
I upgraded Xamarin.Facebook.LoginKit.iOS and Xamarin.Facebook.CoreKit.iOS to verion 5.6.0.1 (it used to be 4.x). After upgrade, nothing happens when I press Login button on iOS app.

I have Xamarin.Forms app, the button is created like this:

```
               var control = new LoginButton();
                //control.LoginBehavior = LoginBehavior.Native;
                control.Permissions = new string[] { "public_profile" };

```
I see in the logs 
`2019-10-25 12:01:08.730 TennisUmpireXamariniOS[445:4801321] -canOpenURL: failed for URL: "fbauth2:/" - error: "The operation couldn’t be completed. (OSStatus error -10814.)"
`
But based on what I could find, it should not be the reason. 

Here is from my Info.plist (didn't change)

```
    <key>LSApplicationQueriesSchemes</key>
    <array>
      <string>fbapi</string>
      <string>fbauth2</string>
      <string>fb-messenger-api</string>
    </array>
```

Is there anything else needs to be done as part of upgrade? Did I miss something?

Thank you for the help.

**Comment by SotoiGhost** [comment]
@maxal1917 This error is fixed in version 5.6.0 as stated in issue #143. Please, clean your NuGet cache, delete your bin and obj folders and try again.

**Comment by SotoiGhost** [comment]
We have not received the requested information. If you are still experiencing this issue, please, open a new issue related to this. Thanks!
